src/styles/abstract/mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin font($size, $weight: 400, $height: normal, $style: normal) {
+  font-size: $size;
+  line-height: $height;
+  font-weight: $weight;
+  font-style: $style;
+}
+
+@mixin border($size, $color, $radius: none) {
```

Comment:
The default value for `$radius` should be `0` instead of `none` to ensure proper CSS syntax, as `none` is not a valid value for `border-radius`.